src/components/RecipeForm/RecipeForm.js

```diff
@@ -0,0 +1,101 @@
+import {nanoid} from 'nanoid';
+import {useState, useEffect} from 'react';
+import {useNavigate} from 'react-router-dom';
+import styled from 'styled-components';
+
+import {getFromLocal, setToLocal} from '../../lib/localStorage.js';
+import IngredientsForm from '../IngredientsForm/IngredientsForm';
+
+export default function RecipeForm() {
+  const navigate = useNavigate();
+  const [ingredients, setIngredients] = useState([]);
+  const [recipes, setRecipes] = useState(getFromLocal('recipes') ?? []);
+
+  function addIngredients(ingredient) {
+    setIngredients([...ingredients, ingredient]);
+  }
+
+  function addRecipe(recipe) {
+    setRecipes([...recipes, recipe]);
+  }
+
+  useEffect(() => setToLocal('recipes', recipes), [recipes]);
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const form = event.target;
+    const {name, prepTime, category} = form.elements;
+    const recipe = {id: nanoid(), name: name.value, prepTime: prepTime.value, category: category.value, ingredients};
+    addRecipe(recipe);
+    setIngredients([]);
+
+    setToLocal('recipes', [...recipes, recipe]);
+    console.log(recipes);
+    console.log(recipe);
+
+    navigate('/recipes/' + recipe.id);
+    form.reset();
+    name.focus();
+  }
+
+  return (
+    <CreateForm onSubmit={handleSubmit}>
+      <CreateLabel htmlFor="name">Add name:</CreateLabel>
+      <input id="name" name="name" placeholder="Thai curry soup" />
+
+      <CreateLabel htmlFor="prepTime">
+        Add prepTime<small>(mins)</small>:
+      </CreateLabel>
+      <input type="number" id="prepTime" name="prepTime" placeholder="30" />
+
+      <CreateLabel htmlFor="category">Select category:</CreateLabel>
+      <CreateSelect id="category" name="category">
+        <option value="Vegetarian">Vegetarian</option>
+        <option value="Fish">Fish</option>
+        <option value="Meat">Meat</option>
+        <option value="Dessert">Dessert</option>
+      </CreateSelect>
+
+      <br></br>
+      <IngredientsForm onAddIngredients={addIngredients} />
+      <br></br>
```

Comment:
Please use CSS to add break lines and no extra HTML elements :)
```suggestion
      <IngredientsForm onAddIngredients={addIngredients} />
```